support-docs/README.md

```diff
@@ -0,0 +1,107 @@
+1	Introduction 
+As the demand for hardware design automation tools increases, there is a need for machine readable datasheets. 
+Defining a common digital datasheet specification reduces the burden for component vendors to deliver multiple datasheets to support different tools and encourages reuse of tools across multiple designs. 
+
+1.1	Objectives
+A part manufacturer creates a datasheet to document the information users need to know about a component. This information can include the performance, electrical characteristics, size, orientation, packaging, and order details.  A digital datasheet is a machine-readable version of the datasheet. The objective of this specification is to standardize the format and the rules used to create digital datasheets. 
+
+1.2	Scope
+This document targets digital datasheet producers including components vendors and tools implementers who are parsing existing datasheets and converting them into machine readable formats. This document is also intended for tool implementers who will consume these digital datasheets to automate designs. The specification will include common classes of components used in designs. Some examples are included in the appendices to help the users better understand how to implement the specification. 

```

Comment:
This document targets digital datasheet producers including components vendors and tools implementers who are parsing existing datasheets and converting them into machine readable formats. This document is also intended for tool implementers who will consume these digital datasheets to automate designs.

suggest:

This document is intended for digital datasheet producers such as components vendors as well as people who will consume these digital datasheets to automate designs such as tool creators.

Reply:
Changed